### DISCUSSION TITLE
Smart connections local AI

### DISCUSSION THREAD

**Discussion by magnushinge** [brianpetro/obsidian-smart-connections#418] [Ideas]
I really hope smart connections get LM server intergration so I can use localAI. Obsidian should be able to run offline imo.

**Comment by brianpetro** [comment]
@MacBeef thanks for the feedback 😃

V2 has a local embedding model. So the Smart View feature will work offline. Still working on the local chat model though

🌴

**Reply by magnushinge** [reply]
Ah I see, does that only work with chatgpt however?

**Reply by brianpetro** [reply]
I'm not sure what you mean. The Smart View is in obsidian and shows similar notes to your current note.

**Reply by Natedude** [reply]
If you pay $120 for V2 privacy with embeddings, does this mean you still have to send private embedding info to OpenAI for the local chat if you don't have the system requirements to run a local llm?

**Reply by brianpetro** [reply]
@Natedude 

What would still be sent to OpenAI are the segments of notes used for context, which is quite limited compared to sending them your entire vault for embedding.

There are also exclusion settings which I recommend using to exclude your most sensitive notes from being used by Smart Connections.

Re:
> If you pay $120 for V2 privacy with embeddings

Regarding this, supporting the project allows benefits like early access to V2, which does have access to a local embedding model. But, becoming a supporter is not the same as buying some sort of service that offers privacy guarantees. Furthermore (from https://github.com/brianpetro/obsidian-smart-connections/discussions/264#discussioncomment-8191265):

> To clarify my intentions, the supporter option isn't meant to be a product, but a way for people that regularly utilize Smart Connections to support the project. The extras that are offered, which currently include the experimental ChatGPT integrations, are things that either I can't ship to everyone at once, like early releases that would have too many issues for a general audience, and things that have a direct cost for me to provide, like the ChatGPT integrations which require a dedicated server, time spent managing the server, etc.

> I don't spend much time promoting the supporter option because I believe it's best for me to focus my time on contributing to other aspects of the project, and also in hope that the minimal nature of the offering only attracts people that truly want to support the project rather than see "supporting" as some sort of alternative name for a service they're paying for, which would imply many more obligations that could quickly make the whole proposition no longer worthwhile from my perspective.

Thanks for your interest in Smart Connections
🌴
